src/World.java

```diff
@@ -92,10 +88,34 @@ else if(i==1) {
         return theCitiesOnMap;
     }
 
+    //NOTE: For now, keeping it simple.  Just random.  Might want to re-balance payout later.
+    public CargoOrder getCargoOrder() {
+        //Chose a random product, city and payout.
+        Random random = new Random();
+        Product chosenProduct = Product.valueOf(Product.getRandom().name());
+        City chosenCity = cities[random.nextInt(cities.length)];
+        int chosenPayout = random.nextInt(20,50);
+
+        //Create and return the CargoOrder.
+        CargoOrder cargoOrder = new CargoOrder(chosenProduct, chosenCity, chosenPayout);
+        return cargoOrder;
+    }
```

Comment:
As much of this as possible should be moved to the CargoOrder class. I know it can't choose a city on its own, but you can either pick one and pass it, or pass it the whole list and have it pick.

Reply:
OK, done.